TheForceEngine/TFE_FrontEndUI/frontEndUi.cpp

```diff
@@ -3171,6 +3177,7 @@ namespace TFE_FrontEndUI
 				gameSettings->df_showSecretFoundMsg = false;
 				gameSettings->df_bobaFettFacePlayer = false;
 				gameSettings->df_smoothVUEs = false;
+				gameSettings->df_solidWallFlagFix = false;
```

Comment:
Same as the other comment, default to true if it doesn't break mods so people get the best experience. Most people won't ever change these settings.

Reply:
What I've done here is leave it as `false` for vanilla mode, and set it to `true` in modern mode

Reply:
Sounds good, thanks.